calculator.go

```diff
@@ -1,13 +1,96 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"go/token"
+	"go/types"
+	"math"
+	"strconv"
+)
+
+const divideByZeroMessage = "Cannot divide by zero"
```

Comment:
It's going to be more convenient for the reader if we simply reference this string where it's used, rather than defining it as a constant. (A string literal is already a constant, if you think about it!)

Otherwise the reader has to see the name of the constant and then trace through the file to find out where it's actually defined, what its value is, and then find her way back to the place where it was used, if possible. That's a cognitive roadblock we don't need.